release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -627,11 +627,13 @@ For more information, see xref:../openshift_images/image-configuration.adoc#imag
 
 You must enable the `TechPreviewNoUpgrade` feature set in the `FeatureGate` custom resource (CR) to enable the `imageStreamImportMode` feature. For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling-features-about_nodes-cluster-enabling[Understanding feature gates].
 
+////
 [id="ocp-release-notes-olm_{context}"]
 === Operator lifecycle
 
 [id="ocp-release-notes-osdk_{context}"]
 === Operator development
+////
 
 [id="ocp-release-notes-openshift-cli_{context}"]
 === OpenShift CLI (oc)
```

Comment:
@dfitzmau All of the O's are out of alphabetical order. They should be moved down below the N's.

Reply:
Thanks. Updated!